app/frontend/src/store/form.js

```diff
@@ -8,9 +8,11 @@ import {
   rbacService,
   userService,
 } from '~/services';
+import { useAppStore } from '~/store/app';
 import { useNotificationStore } from '~/store/notification';
 import { IdentityMode, NotificationTypes } from '~/utils/constants';
 import { generateIdps, parseIdps } from '~/utils/transformUtils';
+import { encryptionKeyService, eventStreamConfigService } from '../services';
```

Comment:
We have an alias that gets you to the root of the front end src path. This would just look like `import { encryptionKeyService, eventStreamConfigService } from '~/services';`

Reply:
awesome, i'll do that. i'll see if I have the same code in other places and address it.